packages/web-pkg/src/components/SideBar/Spaces/Details/SpaceDetails.vue

```diff
@@ -167,16 +142,36 @@ export default defineComponent({
       )}`
     })
 
+    watch(
+      () => unref(resource).spaceImageData,
+      async () => {
+        if (!unref(resource).spaceImageData) {
+          return
+        }
+
+        const imageResource = buildSpaceImageResource(unref(resource))
+        spaceImage.value = await loadPreview({
+          space: unref(resource),
+          resource: imageResource,
+          dimensions: ImageDimension.Tile,
+          processor: ProcessorType.enum.fit,
+          cancelRunning: true,
```

Comment:
Really an edge case, but couldn't it happen now that you show space details from the space overview while previews are still loading and then that would kill all the remaining preview requests in the space overview?

Reply:
`cancelRunning` would only cancel preview requests within this component. The space overview, while also using the `useLoadPreview` composable, has a different "instance" of it. So technically there are 2 separate tasks.

Reply:
Aha! nice, thanks for the explanation 👍